make/targets/golang/imagebuilder.mk

```diff
@@ -0,0 +1,20 @@
+self_dir :=$(dir $(lastword $(MAKEFILE_LIST)))
+
+IMAGEBUILDER_VERSION ?=v1.2.1
+
+ensure-imagebuilder:
+ifeq "" "$(shell which imagebuilder)"
+	$(info Installing imagebuilder)
+	go get github.com/openshift/imagebuilder/cmd/imagebuilder@$(IMAGEBUILDER_VERSION)
```

Comment:
Note: I don't love `go get`. It requires such a delicate balance of go versions, variables, and dependencies. If there's a better way to do this, I'm all ears.

Reply:
One of the reasons we decided for now not to install this automatically. If you look at other places, we download binaries. At most I'm inclined to add a check if imagebuilder exists and if not inform users how to get it.